src/lib.rs

```diff
@@ -8,13 +8,13 @@ extern crate io_lifetimes;
 extern crate libc;
 pub extern crate libudev_sys as ffi;
 #[cfg(feature = "mio06")]
-extern crate mio06;
+pub extern crate mio06;
```

Comment:
Why is mio06 special cased here? Should that be also exported as "mio"?

Reply:
Because the API is different, see the other changes in `monitor.rs`.
I wanted to make that clear since every other version after that have the same API.